src/vl-input-addon.js

```diff
@@ -22,4 +35,20 @@ export const VlInputAddonElement = (SuperClass) => {
  */
 export class VlInputAddon extends VlInputAddonElement(HTMLParagraphElement) {}
 
-define('vl-input-addon', VlInputAddon, {extends: 'p'});
\ No newline at end of file
+/**
+ * VlButton-Input-addon
+ * @class
+ * @classdesc Gebruik de button-input-addon in combinatie met de vl-ui-input-group webcomponent. Deze combinatie maakt de input-addon clickable.
```

Comment:
Ik zou er van maken:
```suggestion
 * @classdesc Gebruik de vl-button-input-addon in combinatie met de vl-input-group webcomponent. Deze combinatie zorgt er voor dat men een button heeft die naast de input in vl-input-group staat.
```